src/vec8bit.c

```diff
@@ -5733,7 +5732,7 @@ Obj FuncTRANSPOSED_MAT8BIT( Obj self, Obj mat)
                         val = vals[0];
 
                     /* set entry */
-                    ptr = BYTES_VEC8BIT(ELM_GF2MAT(tra, nstart + j)) + imod;
+                    ptr = BYTES_VEC8BIT(ELM_MAT8BIT(tra, nstart + j)) + imod;
```

Comment:
Then line 4036 (in this commit) can also be changed from using `SET_ELM_GF2MAT` to `SET_ELM_MAT8BIT`.